joml-geometry/src/main/java/org/terasology/joml/geom/Rectanglef.java

```diff
@@ -394,11 +419,9 @@ public Rectanglef intersection(Rectanglef other) {
     /**
      * Compute the rectangle of intersection between <code>this</code> and the given rectangle.
      * <p>
-     * If the two rectangles do not intersect, then {@link Float#NaN} is stored in each component
-     * of <code>dest</code>.
+     * If the two rectangles do not intersect, then {@link Float#NaN} is stored in each component of <code>dest</code>.
```

Comment:
There is no `dest` is there?

Reply:
same here (unrelated formatting change).